pkg/event/model.go

```diff
@@ -15,19 +14,19 @@ type ResourceEventModel interface {
 	GetSource() string
 	GetResourceType() string
 	GetResourceId() string
-	GetMeta() database.NullString
+	getMeta() *string
```

Comment:
@efirs , do you know if a pointer to a type (e.g. `*string`) has similar marshal/unmarshal behavior to a `sql.NullX` type (i.e. [`sql.NullString`](https://pkg.go.dev/database/sql#NullString))?

I think your change to use pointers in place of our `database.NullX` structs in the Spec and Model structs is correct so that we aren't assuming the eventstore is backed by a SQL database, but I just want to make sure that the behavior is the same.

Reply:
Right, the behavior is the same. But pointer is more idiomatic, IMO.